hack/create-cluster.sh

```diff
@@ -20,15 +20,30 @@ if [[ -z "${CLUSTER_NAME}" ]]; then
   usage
   exit 1
 fi
-KUBECONFIG=$2
+if [[ -z "${2-}" ]]; then
+  KUBECONFIG=$KUBECONFIG
+else
+  KUBECONFIG=$2
+fi
+
 if [[ -z "${KUBECONFIG}" ]]; then
   usage
   exit 1
 fi
 
+# check kind
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# shellcheck source=util.sh
+source "${REPO_ROOT}"/hack/util.sh
+util::cmd_must_exist "kind"
+
 if [ -f "${KUBECONFIG}" ];then
-  echo "Removing old kubeconfig file."
-  rm -f ${KUBECONFIG}
+  echo "kubeconfig file is existed, new config will append to it."
+  if kubectl config get-contexts "${CLUSTER_NAME}" --kubeconfig="${KUBECONFIG}"> /dev/null 2>&1;
+  then
+    echo "ERROR: failed to create new cluster for context '${CLUSTER_NAME}' existed in ${KUBECONFIG}. please remove it (use 'kubectl config delete-context') if your want to recover it."
```

Comment:
> if your want to recover it.

Just a question, what do you mean `recover`?

Reply:
sorry, typo. it should be 'cover'